### PR TITLE
Plane: cleanup geofence warnings

### PR DESCRIPTION
**Pull request by magicrub**
when trying to use the Plane geofence I would get a single error string which wasn't very useful. This PR adds more strings describing the different errors.

### CODE REVIEW

**Comment by khancyr** [comment]
LGTM ! 
maybe you can make a function with the error string as param to prevent the code duplication and easier maintenance. But it already looks better than before !

File: ArduPlane/geofence.cpp [hunk 1]
```diff
@@ -91,24 +91,31 @@ void Plane::set_fence_point_with_index(Vector2l &point, unsigned i)
...
     uint8_t i;
+    const char* geofence_error_str = "Geofence error,";
 
     if (geofence_state == nullptr) {
         uint16_t boundary_size = sizeof(Vector2l) * max_fencepoints();
         if (hal.util->available_memory() < 100 + boundary_size + sizeof(struct GeofenceState)) {
             // too risky to enable as we could run out of stack
-            goto failed;
+            g.fence_action.set(FENCE_ACTION_NONE);
+            gcs().send_text(MAV_SEVERITY_WARNING,"%s low on memory", geofence_error_str);
```
Comment (khancyr) [inline ArduPlane/geofence.cpp]: miss a very important space after warning comma ! (missing on all warning)

Comment (WickedShell) [reply ArduPlane/geofence.cpp]: Would be better to put the space in the `geofence_error_str` as it's then common :)

**Comment by magicrub** [comment]
I had the space up in the common string but the %s up against the strings caught me twice so I figured it was better trade off of 4 bytes over human error while maintaining.

**Comment by magicrub** [comment]
I'll move the space up top

**Pull request merged**